### PR TITLE
Changelog 1.9.5 (#8753)

### PR DESCRIPTION
**Pull request by jolheiser**
Frontport to `release/v1.10.0`

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: CHANGELOG.md
Comment (jaqra) [inline CHANGELOG.md:313]: Sorry for comment after approve but is this section needed?

Comment (jolheiser) [reply CHANGELOG.md:313]: I'll leave it up to others. It was on the `release/v1.9.5` changelog.  
I know we decided to remove them from the master release changelogs, but I don't think we have a concrete strategy for when to keep/remove them. (Or at least I'm not aware of one)

I'm fine with it either way. 😄

**Review by zeripath — APPROVED** [review]

**Pull request merged**
